services/src/main/scala/cromwell/services/cost/GcpCostCatalogTypes.scala

```diff
@@ -94,28 +121,30 @@ object MachineCustomization {
    - For non-N1 machines, both custom and predefined SKUs are included, custom ones include "Custom" in their description
    strings and predefined SKUs are only identifiable by the absence of "Custom."
    */
-  def fromSku(sku: Sku): Option[MachineCustomization] = {
+  def fromCpuOrRamSku(sku: Sku): MachineCustomization = {
```

Comment:
It seemed unnecessary to have this method return an `Option` when the returned value is always a `Some`, especially when that made the return type inconsistent with `fromMachineTypeString`. This method should not be used for GPU SKUs.